### DISCUSSION TITLE
SingleFile Not Always Downloading All Content In zip

### DISCUSSION THREAD

**Discussion by rideoutn** [gildas-lormeau/SingleFile#1429] [Q&A]
New to using this extension, I've fiddled with the settings and changed it so it downloads pages as a .zip with the HTML file, images, fonts, scripts, etc, but I keep getting this recurring problem where it will download the page but when I open the zip there will only be a folder such as "Fonts" or "Images" inside. When this happens if I try again a few times EVENTUALLY the zip will have all the files, but it's kind of annoying. Is there just an option I didn't activate that would make it wait until everything is downloaded before creating the zip file? I've been primarily using it in Firefox, but I tried installing the extension in other apps with the same settings and I still get the problem.

**Comment by rideoutn** [comment]
Never mind. I did some digging when I realized a zip was 70mb in size but still only had one font file in it. For reasons I can't figure out SOMETIMES opening the zip file in 7Zip only shows the one folder, but if I instead open it in another archive program like WinRAR I can see all the files. Probably just a weird problem on my end.

**Reply by gildas-lormeau** [reply]
Thank you very much for the feedback This might be a bug in zip.js. I'm also the author of this library, so I could maybe fix it. Could you attach a zip file with this issue?
